src/Umbraco.Configuration/GlobalSettings.cs

```diff
@@ -227,7 +214,7 @@ public string ConfigurationStatus
             }
             set
             {
-                SaveSetting(Constants.AppSettings.ConfigurationStatus, value, _ioHelper);
+                SaveSetting(Constants.AppSettings.ConfigurationStatus, value, Current.IOHelper); //TODO remove
```

Comment:
Is this TODO for later use?

Reply:
This is a class that will be removed in the future.. I thing the TODO is fine for now